README.md

```diff
@@ -1,2 +1,70 @@
-# math-magicians
-Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a random math-related quote.
+# boiler-templet
+
+## MATH MAGICIANS
+
+Creating my Calculator.
+
+# Math Magicians
+> This is the first project of the Math Magicians application. You will set up the environment and tools needed to develop a React application. In the following projects, you will develop the actual application.
+
+## Built With
+-goolge
+- HTML5
+- CSS
+- Javascript
+- React
+
+## How to use and run this project
+
+>To get a local copy up and running follow these simple example steps.
+
+- Clone this repository with
+
+git clone https://github.com/zairdon20/First-react.git using your terminal or command line.
+
+- Change to the project directory by entering :
+
+cd todo-list-project in the terminal.
+
+- run code . to open it in vscode
+- Install packages using the command npm install
+- run the project locally by entering the command npm start
+
+
+
+## Project Status
+This project is in progress.
+
+## Screenshoots
+![screenshot](src\assets\calculatorpic.png)
```

Comment:
- [ ] **[OPTIONAL]** I noticed that the link to this screenshot is broken. You can decide to remove this broken link or try fixing it to display the screenshot properly. 😉.